src/oryx/install.sh

```diff
@@ -156,20 +156,17 @@ PATCH_VERSION_ID=$(echo $(dotnet --version) | cut -d . -f 3)
 
 PINNED_SDK_VERSION=""
 # Oryx needs to be built with .NET 8
-if [[ "${DOTNET_BINARY}" = "" ]] || [[ $MAJOR_VERSION_ID != "8" ]] || [[ $MAJOR_VERSION_ID = "8" && ${PATCH_VERSION_ID} -ge "101" ]] ; then
+if [[ "${DOTNET_BINARY}" = "" ]] || [[ $MAJOR_VERSION_ID != "8" ]] || [[ $MAJOR_VERSION_ID = "8" && ${PATCH_VERSION_ID} -ne "202" ]] ; then
```

Comment:
What about patch below and above 202?

Reply:
**If Dotnet SDK PATCH version is below 202 Oryx build fails**: 
mcr.microsoft.com/devcontainers/base:noble image do not have dotnet installed.
When installed dotnet using apt it will install **dotnet 8.0.110** and Oryx build fails for this version.
**If Dotnet SDK PATCH version is 202 Oryx build works**: 
So to be consistent we are installing .net sdk 8.0.202 if this version is not in image
In existing code this installed sdk for Oryx is removed after oryx build completes. So it would not affect any other dotnet behavior of the image